packages/components/src/sortTabs/SortTabs.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { StyleSheet } from '@kiwicom/universal-components';
+
+import SortTab from './SortTab';
+
+export type SortIconTypes = 'thumb-up' | 'money' | 'clock';
+export type SortTypes = 'QUALITY' | 'PRICE' | 'DURATION';
+type SortTabsDataType = $ReadOnlyArray<{|
+  +label: string,
+  +value: SortTypes,
+  +icon: SortIconTypes,
+|}>;
+
+export type Props = {|
+  +selectedValue: ?SortTypes,
+  +onValueChange: (?SortTypes) => void,
+|};
+
+export const SortTabsData: SortTabsDataType = [
+  { label: 'Best', value: 'QUALITY', icon: 'thumb-up' },
+  { label: 'Cheapest', value: 'PRICE', icon: 'money' },
+  { label: 'Fastest', value: 'DURATION', icon: 'clock' },
+];
+
+const SortTabs = ({ selectedValue, onValueChange }: Props) => {
+  const SortTabs = SortTabsData.map((SortTabData, index) => {
```

Comment:
Can you rename `SortTabs`? There are two same names and it's not clear.

Reply:
Done! renamed to `SortTabsList`